crates/swc_ecma_minifier/tests/fixture/issues/8465/output.js

```diff
@@ -0,0 +1,4 @@
+function Infinity() {
+    console.log("xxx");
+}
+export default Infinity;
```

Comment:
This doesn't seem like to be the optimal output. It should be
```js
export default function() {
    console.log("xxx");
};
```

Reply:
check this https://github.com/swc-project/swc/pull/8469/commits/1aefb0a7ccca8ea15ebb135c28ce88a8973adbf9#diff-0c587eaeb38866a706c0a8355bf41885ef6871afd928e60a5975fe1a46eb0d1fR1-R3